templates/easy_breadcrumb/easy-breadcrumb.tpl.php

```diff
@@ -5,11 +5,9 @@
  * Contains template file.
  */
 ?>
-<div style="background-color: #004494; padding: 5px 15px 15px; float: left; width: 100%;">
-<nav id="breadcrumb" class="breadcrumb" role="navigation" aria-label="breadcrumbs">
-  <span class="element-invisible"><?php echo t('You are here'); ?></span>
-  <ol class="breadcrumb__segments-wrapper">
+<nav class="ecl-breadcrumbs <?php print render($extra_class); ?>" aria-label="breadcrumbs">
```

Comment:
Here in fact, there is no need in this case, to create an extra variable for this. Furthermore, you may just use the `$attributes` variables because it is already there.